spec/models/tattoo_spec.rb

```diff
@@ -3,7 +3,8 @@
 RSpec.describe Tattoo, type: :model do
 
   context 'tattoos' do
-    let!(:tattoo) { FactoryGirl.create(:tattoo) }
+    let!(:tattoo) { FactoryGirl.create(:tattoo,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.